src/components/organisms/ActionsPane/ActionsPane.tsx

```diff
@@ -317,13 +317,14 @@ const ActionsPane = (props: {contentHeight: string}) => {
   }, [selectedResourceId, resourceMap]);
 
   useEffect(() => {
-    if (
-      (activeTabKey === 'metadataForm' || activeTabKey === 'form') &&
-      (!selectedResourceId || !(isKustomization || (resourceKindHandler && resourceKindHandler.formEditorOptions)))
-    ) {
+    if (activeTabKey === 'form' && !(resourceKindHandler && resourceKindHandler.formEditorOptions?.editorSchema)) {
```

Comment:
need an isKustomization check here - which won't have a resourceKindHandler but is still valid for the form tab

Reply:
resolved